src/new-client/src/services/currencyPairs.ts

```diff
@@ -104,5 +105,9 @@ export {
   useFilteredSymbols,
   useCurrencies,
   onSelectCurrency,
```

Comment:
Should we standardize on always exporting the whole bind tuple and a naming convention for the export?  e.g., `export const currencyPairRxBinding = bind(...)` - @josepot ?

Reply:
This would be for a later PR, just initiating the discussion.